bin/rucio-reaper2

```diff
@@ -60,6 +61,8 @@ def get_parser():
                         help='Number of service unavailable exceptions after which the RSE gets temporarily excluded.')
     parser.add_argument("--auto_exclude_timeout", action="store", default=600, type=int,
                         help='Timeout for temporarily excluded RSEs.')
+    parser.add_argument("--bulk", action="store_true", default=False,
```

Comment:
Is this only valid for Globus deletion? If so, maybe this should be renamed to something like --globus-bulk, or alternatively change the comment "Enable bulk deletion in Globus". Because otherwise I would even go as far as recommending this to be set to default=true.

Reply:
Also, might be worth to actually make this an integer instead of a bool, to set the bulk size?

Reply:
Yes this is to bundle multiple file deletes into 1 async job for Globus.  I like the idea to make this an integer because that removes the dependency on `chunk_size`.  Your idea to make this generic is also a good one.  Another service could easily use the reaper to do the same.